test/EntityFramework6.Npgsql.Tests/EntityFrameworkBasicTests.cs

```diff
@@ -772,5 +772,67 @@ public void TestTableValuedStoredFunctions()
                 Assert.AreEqual(1, list2[0].Something);
             }
         }
+
+        [Test]
+        public void Test_issue_60_and_62()
```

Comment:
This test needs a better name, maybe: Test_string_type_inference_in_coalesce_statements